src/Files.App/Commands/HotKey/HotKey.cs

```diff
@@ -1,38 +1,301 @@
-﻿using System;
+﻿using Files.App.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Forms = System.Windows.Forms;
 
 namespace Files.App.Commands
 {
+	[DebuggerDisplay("{Code}")]
 	public readonly struct HotKey : IEquatable<HotKey>
 	{
+		private static readonly IImmutableDictionary<KeyModifiers, string> modifiers = new Dictionary<KeyModifiers, string>()
+		{
+			[KeyModifiers.Menu] = GetKeyString("Menu"),
+			[KeyModifiers.Ctrl] = GetKeyString("Control"),
+			[KeyModifiers.Shift] = GetKeyString("Shift"),
+			[KeyModifiers.Win] = GetKeyString("Windows"),
+		}.ToImmutableDictionary();
+
+		private static readonly IImmutableDictionary<Keys, string> keys = new Dictionary<Keys, string>()
```

Comment:
This doesn't feel right to me. Maybe you could iterate over the enum values and call `.ToString()` on them

Reply:
These texts are localized because the keys do not have the same name in all languages.

Reply:
ehhh